app/javascript/article-form/components/__stories__/TagAutocompleteOption.stories.jsx

```diff
@@ -1,5 +1,5 @@
 import { h } from 'preact';
-import { TagAutocompleteOption } from '../TagAutocompleteOption';
+import { TagAutocompleteOption } from '../../../crayons/MultiSelectAutocomplete/TagAutocompleteOption';
```

Comment:
```suggestion
import { TagAutocompleteOption } from '@crayons/MultiSelectAutocomplete/TagAutocompleteOption';
```

(not blocking) - just another place we could use that alias